test/e2e/grpc/client.test.ts

```diff
@@ -138,7 +138,7 @@ describe("grpc/client", () => {
     // it('should be able to get the capabilities of components via metadata call', async () => {
     //   const res = await client.metadata.get();
     //   const redisStateComponent = res.components.filter( (component) => component.name == "state-redis" );
-    //   const expectedRedisStateCapabilities = [ 'ETAG', 'TRANSACTIONAL', 'QUERY_API', 'ACTOR' ];
+    //   const expectedRedisStateCapabilities = [ 'ETAG', 'TRANSACTIONAL', 'ACTOR' ];
```

Comment:
The Redis component changed and now doesn't report QUERY_API unless redisjson is being used. The test needs updating